dev/packages/beats/apache-0.0.1/manifest.yml

```diff
@@ -9,6 +9,7 @@ categories:
 - logs
 - metrics
 release: ga
+removable: false
```

Comment:
This should be true as the default is true. I suspect an error in the package code?

Reply:
Fixed.